modules/enabling-habana-gaudi-devices.adoc

```diff
@@ -4,53 +4,51 @@
 = Enabling Habana Gaudi devices
 
 [role='_abstract']
-Before you can use Habana Gaudi devices in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator in your deployment. 
+Before you can use Habana Gaudi devices in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator. 
 
 .Prerequisites
 * You have logged in to {openshift-platform}.
 * You have the `cluster-admin` role in {openshift-platform}.
 
 .Procedure
 . To enable Habana Gaudi devices in {productname-short}, follow the instructions at link:https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator for OpenShift].
-. Create an accelerator profile for your Habana Gaudi device. 
-.. In the {openshift-platform} web console in the *Administrator* perspective, click *Administration* -> *CustomResourceDefinitions*.
-.. In the search bar, enter `acceleratorprofile` to search by name.
+. From the {productname-short} dashboard, click *Settings* -> *Accelerator profiles*.
 +
-The *CustomResourceDefinitions* page reloads to display the search results.
+The *Accelerator profiles* page appears that shows existing accelerator profiles. To enable or disable an existing accelerator profile, on the row containing the relevant accelerator profile, click the toggle in the *Enable* column.
```

Comment:
Same as previous, suggest adjusting to 'appears, displaying existing accelerator profiles'.